src/app/components/generics/property-input/select-input.component.html

```diff
@@ -1,19 +1,19 @@
 <mat-form-field>
   <mat-label class="input-label">{{ name }}</mat-label>
-  <mat-select *ngIf="isBeingEdited" [(ngModel)]="value">
+  <mat-select [(ngModel)]="inputValue" (ngModelChange)="inputChanged()">
```

Comment:
is there a reason to use the property [(ngModel)] instead of [value] (as in [src/app/components/execution-environments/cloud-services/cloud-service-properties/cloud-service-properties.component.html](url) or other html files)